### DISCUSSION TITLE
Mismatch between forkCount and totalCount

### DISCUSSION THREAD

**Discussion by just-boris** [community/community#33375] [Apps, API and Webhooks] [answered]
When I am making the following query

```graphql
query {
  repository(owner: "cloudscape-design", name: "components") {
    forkCount
    forks(last: 1) {
      totalCount
    }
    stargazerCount
    stargazers(last:1) {
      totalCount
    }
  }
}
```

I get the following response

```json
{
  "data": {
    "repository": {
      "forkCount": 74,
      "forks": {
        "totalCount": 56
      },
      "stargazerCount": 1685,
      "stargazers": {
        "totalCount": 1685
      }
    }
  }
}
```

`forkCount` and `forks.totalCount` should be the same value, but for some reason it is not.

This is only for the forks count. Stars numbers match.

P.S. There also was [a stackoverflow question](https://stackoverflow.com/questions/72551536/github-graphql-get-all-forks-of-a-repository) about the same issue, but no answers there

**Comment by airtower-luna** [comment, marked as answer]
The [documentation of the "repository" resource](https://docs.github.com/en/graphql/reference/objects#repository) explains it. To quote the descriptions:

* `forkCount`
  > Returns how many forks there are of this repository in the whole network.
* `forks`
  > A list of direct forked repositories

So if someone forks a fork it'll add to the `forkCount`, but not increase the count of `forks`. :sweat_smile:
